src/SFA.DAS.EmployerIncentives.Api/HealthChecks/CommitmentsV2ApiHealthCheck.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace SFA.DAS.EmployerIncentives.Api.HealthChecks
+{
+    public class CommitmentsV2ApiHealthCheck : IHealthCheck
+    {
+        public Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = new CancellationToken())
```

Comment:
Doesn't seem very useful this healthcheck

Reply:
same as above

Reply:
fixed